.github/workflows/docs.yaml

```diff
@@ -5,14 +5,6 @@ on:
   push:
     branches:
       - main
```

Comment:
It's unfortunate that this will mean we're going to regenerate the license report and the database schema diagrams for every commit. Probably not worth optimizing for, though; it just means more GitHub Actions minutes but if we start caring about conserving those, there are much bigger fish to fry than this.